src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1910,6 +1920,12 @@ theorem shiftLeft_add_distrib {x y : BitVec w} {n : Nat} :
   case succ n ih =>
     simp [ih, toNat_eq, Nat.shiftLeft_eq, ← Nat.add_mul]
 
+@[simp]
+theorem add_eq_xor (a b : BitVec 1) : a + b = a ^^^ b := by
```

Comment:
```suggestion
theorem add_eq_xor (a b : BitVec 1) : a + b = a ^^^ b := by
```

Reply:
@hargoniX suggested that simp is likely not a good idea here as the rewrite may not always be more canonical. I agree, let's drop the `@[simp]` for now.